frontend/src/components/ui/config-details.ts

```diff
@@ -259,6 +259,12 @@ export class ConfigDetails extends LiteElement {
                 ISO6391.getName(crawlConfig.config.lang),
               )
             : nothing}
+          ${crawlConfig?.proxyId
+            ? this.renderSetting(
+                msg("SSH Proxy Server"),
```

Comment:
```suggestion
                msg("Proxy"),
```

^ Maybe @Shrinks99 has good terminology ideas here? Best to keep this simple and consistent I think, we don't mention SSH elsewhere and seems like an implementation detail we don't need to communicate to users here.

Reply:
We also use `Crawler Proxy Server` in `select-crawler-proxy.ts`